bug_change_status_page.php

```diff
@@ -238,7 +235,8 @@
 					<?php print_documentation_link( 'due_date' ) ?>
 				</th>
 				<td>
-					<?php echo '<input ' . helper_get_tab_index() . ' type="text" id="due_date" name="due_date" class="datetime" size="20" maxlength="16" value="' . $t_date_to_display . '" />' ?>
+					<?php echo '<input ' . helper_get_tab_index() . ' type="text" id="due_date" name="due_date" class="datetimepicker" size="20" maxlength="16" value="' . $t_date_to_display . '" />' ?>
+					<script type="text/javascript">$( ".datetimepicker" ).datetimepicker({});</script>
```

Comment:
Our CSP policy does not allow inline scripts. This should go to *common.js* or in another, dedicated file.